test/suite/test_tiered02.py

```diff
@@ -27,39 +27,64 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 
 import os, wttest
+from helper_tiered import get_auth_token, get_bucket1_name, get_bucket1_region
+from helper_tiered import generate_s3_prefix
 from wtdataset import SimpleDataSet
+from wtscenario import make_scenarios
 
 # test_tiered02.py
 #    Test tiered tree
 class test_tiered02(wttest.WiredTigerTestCase):
-    uri = "table:test_tiered02"
+    storage_sources = [
+        ('local', dict(ss_name = 'local_store',
+            auth_token = get_auth_token('local_store'),
+            bucket = get_bucket1_name('local_store'),
+            bucket_region = get_bucket1_region('local_store'),
+            bucket_prefix = "pfx_")),
+        ('s3', dict(ss_name = 's3_store',
+            auth_token = get_auth_token('s3_store'),
+            bucket = get_bucket1_name('s3_store'),
+            bucket_region = get_bucket1_region('s3_store'),
+            bucket_prefix = generate_s3_prefix())),
+    ]
+    # Make scenarios for different cloud service providers
+    scenarios = make_scenarios(storage_sources)
 
-    auth_token = "test_token"
-    bucket = "mybucket"
-    bucket_prefix = "pfx_"
-    extension_name = "local_store"
+    uri = "table:test_tiered02"
 
     def conn_config(self):
-        if not os.path.exists(self.bucket):
+        if self.ss_name == 'local_store' and not os.path.exists(self.bucket):
             os.mkdir(self.bucket)
         return \
           'tiered_storage=(auth_token=%s,' % self.auth_token + \
           'bucket=%s,' % self.bucket + \
+          'bucket_region=%s,' % self.bucket_region + \
           'bucket_prefix=%s,' % self.bucket_prefix + \
-          'name=%s),tiered_manager=(wait=0)' % self.extension_name
+          'name=%s),tiered_manager=(wait=0)' % self.ss_name
 
-    # Load the local store extension.
+    # Load the storage store extension.
     def conn_extensions(self, extlist):
+        config = ''
+        # S3 store is built as an optional loadable extension, not all test environments build S3.
+        if self.ss_name == 's3_store':
+            #config = '=(config=\"(verbose=1)\")'
+            extlist.skip_if_missing = True
+        #if self.ss_name == 'local_store':
+            #config = '=(config=\"(verbose=1,delay_ms=200,force_delay=3)\")'
         # Windows doesn't support dynamically loaded extension libraries.
         if os.name == 'nt':
             extlist.skip_if_missing = True
-        extlist.extension('storage_sources', self.extension_name)
+        extlist.extension('storage_sources', self.ss_name + config)
 
     def progress(self, s):
         self.verbose(3, s)
         self.pr(s)
 
     def confirm_flush(self, increase=True):
+        # Todo: Need to do something different for S3, disable for the time being
```

Comment:
Is `Todo:` the right label? If there is a ticket, maybe use `FIXME-WT-<number>` or `TODO:`. I don't know the right answer here.
Also end the sentence comment with a period.

Reply:
FIXME with the correct ticket number is the way to go :)